src/application.py

```diff
@@ -102,6 +103,10 @@ def __init__(self, application_id, **kwargs):
         center_action = settings.get_child("actions").create_action("center")
```

Comment:
As more actions are probably implemented using this way, a loop structure would be useful here. Can be part of a separate pr however